### ISSUE TITLE
sn76489 periodic noise cannot be converted to ay8910

### ISSUE THREAD

**Issue by denjhang** [digital-sound-antiques/vgm-conv#33]
I think it can be regarded as a square wave of the same frequency

**Comment by denjhang** [comment]
![截图未命名0816232218](https://user-images.githubusercontent.com/61811583/129588253-db804c29-f440-4bfe-ad41-174a510367b6.jpeg)

**Comment by okaxaki** [comment]
I think the periodic noise is not like a square wave. vgm-conv >= 0.8.5 converts periodic to white (CH3 freq is also taken into account).

**Comment by denjhang** [comment]
Then you can give an option to choose which periodic noise to convert to.
In fact, in many tracks, the acts as a sawtooth-like bass.

[periodic noise of sn76489.zip](https://github.com/digital-sound-antiques/vgmconv/files/6993608/periodic.noise.of.sn76489.zip)

**Comment by okaxaki** [comment]
Thanks. The sample vgm is very helpful. I understand what you mean. I'll implement an option for that.

**Comment by okaxaki** [comment]
I have added the option `periodicNoiseAssignment`.  It is available from [0.8.7](https://github.com/digital-sound-antiques/vgm-conv/releases/tag/0.8.7). 

```
  -D periodicNoiseAssignment=value   Specify the target to which SN76489's periodic noise will be converted. 
                                     The value must be one of 'tone', 'noise', 'mix' or 'none'. The default value is 'tone'.            
                                     If 'none' is specified, the periodic noise will be muted.
```
